coding-guides/cleverbot-integration.md

```diff
@@ -9,16 +9,16 @@ I've had this request since I started An Idiot's Guide, in fact it was one of th
 So to get started, let's grab the example from [getting started](../getting-started/getting-started-long-version.md) and shove it in a file.
 
 ```javascript
-const Discord = require("discord.js");
-const client = new Discord.Client({
-  intents: ["GUILDS", "GUILD_MESSAGES"]
+const { Client, Intents: { FLAGS: { GUILDS, GUILD_MESSAGES } } } = require("discord.js");
```

Comment:
Honestly, I know what you're doing, but right now, I think we should stick with the _traditional_ format, as we don't (as far as I can remember) explain, or go into detail regarding deconstructing.

Reply:
I'll just stick with destructuring Intents, and using `Intents.FLAGS.GUILDS` for better consistency